src/core/highlight/highlight.view.js

```diff
@@ -116,69 +115,54 @@ export default class HighlightView extends View {
 
 	highlight(key, cls) {
 		const table = this.table;
-		const markup = this.markup;
 		const index = this.columnIndex(key);
 		if (index < 0) {
 			return noop;
 		}
 
 		const head = table.head;
-		if (markup.head.rows.length) {
-			const cells = head.column(index).cells();
-			cells.forEach((cell) => cell.addClass(`${GRID_PREFIX}-${cls}`));
-			if (index > 0) {
-				const cells = head.column(index - 1).cells();
-				cells.forEach((cell) => cell.addClass(`${GRID_PREFIX}-${cls}-prev`));
-			}
-
-			if (index < markup.head.rows.length - 1) {
-				const cells = head.column(index + 1).cells();
-				cells.forEach((cell) => cell.addClass(`${GRID_PREFIX}-${cls}-next`));
-			}
+		const headCells = head.column(index).cells();
+		headCells.forEach((cell) => cell.addClass(`${GRID_PREFIX}-${cls}`));
+		if (index > 0) {
+			const cells = head.column(index - 1).cells();
+			cells.forEach((cell) => cell.addClass(`${GRID_PREFIX}-${cls}-prev`));
 		}
-		if (markup.body.rows.length) {
-			const cells = table.body.column(index).cells();
-			cells.forEach((cell) => cell.addClass(`${GRID_PREFIX}-${cls}`));
-		}
-		if (markup.foot.rows.length) {
-			const cells = table.foot.column(index).cells();
-			cells.forEach((cell) => cell.addClass(`${GRID_PREFIX}-${cls}`));
+
+		if (index < table.head.rowCount() - 1) {
```

Comment:
Is it ```index``` of column? Why here is ```rowCount()```, but not ```columnCount()```?
Besides, minor thing, you had already put ```table.head``` to variable ```head``` and can use it